core/src/main/java/org/apache/accumulo/core/file/rfile/bcfile/PrintInfo.java

```diff
@@ -70,6 +69,6 @@ public static void main(String[] args) throws Exception {
       fs = path.getFileSystem(conf);
     else
       fs = hadoopFs.exists(path) ? hadoopFs : localFs; // fall back to local
-    printMetaBlockInfo(conf, fs, path);
+    printMetaBlockInfo(siteConfig, conf, fs, path);
```

Comment:
We should probably create some sort of "offline context" which is specialized for these cases where we have server-side offline tools which access offline system resources, like site configuration.

Reply:
I like this idea. `UtilityContext` might be a nice name as it's used for Accumulo utilities and not servers.